src/incipit/processing.py

```diff
@@ -128,6 +128,35 @@ def split_regions_by_image(
     return regions_by_image_index
 
 
+def adjust_region_position_by_image(
+        regions: typing.List[typing.Union[Region, IndexedRegion]],
+        expand_vertical_ratio_of_system: float = 0.5,
+        expand_vertical_ratio_of_gap: float = 0.45,
+)-> typing.List[typing.Union[Region, IndexedRegion]]:
+    leftmost_system_boundary = sys.maxsize
+    rightmost_system_boundary = 0
+    previous_system_bottom_y_coordinate = 0
+    index = 0
+    min_vertical_gap_between_systems = sys.maxsize
+    for region in regions:
+        leftmost_system_boundary = min(leftmost_system_boundary, region.x0)
+        rightmost_system_boundary = max(rightmost_system_boundary, region.x1)
+        if( index > 0 ):
+            min_vertical_gap_between_systems = min(region.y0 - previous_system_bottom_y_coordinate, min_vertical_gap_between_systems)
+        previous_system_bottom_y_coordinate = region.y1
+        index += 1
+
+    for region in regions:
```

Comment:
@Allan518 Here, I would put:
```
# In second pass of regions, we normalize regions widths to leftmost and rightmost boundaries
# (FIXME: assumes image is not crooked/skewed; future work should attempt to detect skew and
# limit corrections when too much skew, or fail when too much skew)
```

Reply:
@jlumbroso I have made the change, please review.